Add link to why not use anaconda in install page (#7778)
sunpy/sunpy: docs/tutorial/installation.rst

That's because we will create a new Python environment.
As well as containing a Python installation, this environment provides an isolated place to install Python packages (like ``sunpy``) without affecting any other current Python installation.
If you already have Python and ``conda`` working you can skip the next section.

> `If you are using Anaconda, we recommend that you uninstall it as it has a restrictive license which means you might not be able to use it for free <https://sunpy.org/posts/2024/2024-08-09-anaconda/>`__.

Reviewer: ```suggestion
`If you are using Anaconda, we recommend that you uninstall it as the packages have a license which means you might not be able to use it for free <https://sunpy.org/posts/2024/2024-08-09-anaconda/>`__.
```

Reviewer 2: Or this?

```suggestion
`If you are using Anaconda, we recommend that you uninstall it as the default package channels have a restrictive license which means you might not be able to use it for free <https://sunpy.org/posts/2024/2024-08-09-anaconda/>`__.
```

That's because we will create a new Python environment.
As well as containing a Python installation, this environment provides an isolated place to install Python packages (like ``sunpy``) without affecting any other current Python installation.
If you already have Python and ``conda`` working you can skip the next section.

`If you are using Anaconda, we recommend that you uninstall it as it has a restrictive license which means you might not be able to use it for free <https://sunpy.org/posts/2024/2024-08-09-anaconda/>`__.

> Instead, we recommend that you use miniforge which is a minimal installer for conda which is free to use. If you are using miniforge, you can skip the next section

Author: ```suggestion
Instead, we recommend that you use miniforge which is a minimal installer that setups conda with the conda-forge channel, which is free to use for everyone.
If you are using miniforge, you can skip the next section
```
